tests/integrations/celery/test_celery.py

```diff
@@ -493,17 +492,36 @@ def test_task_headers(celery):
         "sentry-monitor-check-in-id": "123abc",
     }
 
-    @celery.task(name="dummy_task")
-    def dummy_task(x, y):
-        return x + y
-
-    def crons_task_success(sender, **kwargs):
-        headers = _get_headers(sender)
-        assert headers == sentry_crons_setup
```

Comment:
An AssertionError in the signal for some reason didn't actually fail the testcase for me locally, so changed this to check the headers via the result.